auditlog/admin-service/src/main/resources/application.properties

```diff
@@ -0,0 +1,24 @@
+# Configuration file
+# key = value
+
+quarkus.http.port=8085
+
+mp.messaging.incoming.transactions.connector=smallrye-kafka
+mp.messaging.incoming.transactions.bootstrap.servers=localhost:9092
```

Comment:
I was thinking about it and was not sure how to name it... plus it could be slightly misleading for people using MP reactive messaging... so if possible I'd stick to these three props, wdyt?

Reply:
Just to make sure we're on the same page: I was thinking about keeping the three props, but have them reference a single one:

bootstrap.servers=localhost:9092
mp.messaging.incoming.transactions.bootstrap.servers=${bootstrap.servers}
mp.messaging.incoming.vegetables.bootstrap.servers=${bootstrap.servers}
...

Would make it a bit simpler to override with a single value. If that's what you considered and decided against doing, that's cool, too. We can stick to the current approach then.

Reply:
yes, that's what I considered... the thing was that I didn't want to use a property that might get in the way somehow... maybe it's a good thing to have it for MP messaging in general to allow to have properties on the top level to be inherited.

```
mp.messaging.bootstrap.servers=localhost:9092
```

so if you don't mind let's stick to current setup and I will open issue/question for Quarkus.

Reply:
Gotcha. +1 for raising this with Quarkus folks (or mP reactive messaging). Thanks!